server/views/components/find-us/index.njk

```diff
@@ -0,0 +1,11 @@
+<div class="find-us">
+  {% icon 'other/location' %}
+  <p class="find-us__address" itemprop="address" itemscope itemtype="https://schema.org/PostalAddress">
```

Comment:
Nice, but I wonder if we stick this in a [`<script type="application/ld+json">`](https://developers.google.com/schemas/formats/json-ld). <= @davidpmccormick  for reference.